resources/web/server/simulation_server.py

```diff
@@ -249,7 +249,10 @@ def runWebotsInThread(client):
             global config
             world = self.project_instance_path + '/worlds/' + self.world
             port = client.streaming_server_port
-            command = config['webots'] + ' --batch --mode=pause --minimize '
+            command = config['webots'] + ' --batch --mode=pause '
+            # on Windows, the MJPEG stream won't work if the Webots window is minimized
+            if sys.platform != 'win32' or not hasattr(self, 'mode') or self.mode == 'x3d':
```

Comment:
this is the same on linux: the mjpeg streaming only works if the Webots window is not minimized
```suggestion
            if not hasattr(self, 'mode') or self.mode == 'x3d':
```